app/controllers/bookings_controller.rb

```diff
@@ -0,0 +1,42 @@
+class BookingsController < ApplicationController
+    before_action :set_booking, only: [:show, :edit]
+
+    def index
+        @bookings = Booking.all
+    end
+    
+    def new
+        @booking = Booking.new
+    end
+
+    def create
+        @booking = Booking.new(booking_params)
+        @booking.save
+        # TODO: may need to create conditional, for when Car is available or 'not booked'
+        # TODO: add redirect to bookings_path when route created
+    end
+
+    def show; end
+
+    def edit; end
+  
+    def update
+        @booking.update(booking_params)
+        # TODO: add redirect to bookings_path(@booking) when route created
+    end
+  
+    def delete
```

Comment:
I think this should change to destroy

Reply:
the name of the action should be 'destroy'? I thought it was 'delete' and then the method called on @booking was 'destroy'? I could be wrong

Reply:
<img width="732" alt="Screenshot 2021-06-16 at 13 17 15" src="https://user-images.githubusercontent.com/61727046/122161911-2e800680-cea5-11eb-8957-1935d8db265b.png">

I was also unsure. Just checked and it should be destroy.

Reply:
👍🏻